README.md

```diff
@@ -838,6 +838,24 @@ oneview_user 'User1' do
 end
 ```
 
+### oneview_scope
+
+Scope resource for HPE OneView.
+
+Support only in API300 onwards.
+
+```Ruby
+oneview_scope 'Scope1' do
+  client <my_client>
+  data <resource_data>
+  add     # Hash containing combinations of <resourcetype>: <Array of names> to be added to the scope - Optional
+  remove  # Hash containing combinations of <resourcetype>: <Array of names> to be removed from the scope - Optional
+  action [:create, :create_if_missing, :delete, :change_resource_assignments]
+end
+```
+
+- **add** and **remove** (Hash) Optional - Specify resources to be added or removed. The Hashes should have `<resource_type> => [<resource_names>]` associations. See the [example](examples/scope.rb) for more information.
```

Comment:
Specify which type of key you're expecting (Symbol, String) and in what case the `resource_type` should be.